### ISSUE TITLE
Docs migration

### ISSUE THREAD

**Issue by davetapley** [JEFuller/Piexif#2]
https://piexif.readthedocs.io/en/latest/

https://docs.readthedocs.io/en/stable/abandoned-projects.html

**Comment by melonmouse** [comment]
Hey there Dave Tapley, it is super great that you are taking this on!!

Just here to say that it is hard to figure out that your fork is the official one because readthedocs was not taken over yet. As a result you might be missing out on pull requests, volunteers, stars and all other things that help make a project alive!

Best of luck :D

**Comment by davetapley** [comment]
@melonmouse yep, I figured I'd do #1 first in case the original owner was reachable during that process, but alas not.

I just opened: https://github.com/readthedocs/readthedocs.org/issues/9619

**Comment by davetapley** [comment]
Six week wait (assuming https://github.com/hMatoba is still MIA): https://github.com/readthedocs/readthedocs.org/issues/9619#issuecomment-1262569733

In light of this I will start other fork migration tasks before this.
